src/main/java/se/sundsvall/datawarehousereader/api/CustomerResource.java

```diff
@@ -49,7 +50,7 @@ public ResponseEntity<CustomerEngagementResponse> getCustomerEngagements(@Valid
 	@ApiResponse(responseCode = "200", description = "Successful operation", useReturnTypeSchema = true)
 	@ApiResponse(responseCode = "400", description = "Bad request", content = @Content(mediaType = APPLICATION_PROBLEM_JSON_VALUE, schema = @Schema(oneOf = { Problem.class, ConstraintViolationProblem.class })))
 	@ApiResponse(responseCode = "500", description = "Internal Server error", content = @Content(mediaType = APPLICATION_PROBLEM_JSON_VALUE, schema = @Schema(implementation = Problem.class)))
-	public ResponseEntity<CustomerDetailsResponse> getCustomerDetails(@Valid CustomerDetailsParameters searchParams) {
+	public ResponseEntity<CustomerDetailsResponse> getCustomerDetails(@Valid @ValidCustomerDetailsParameters CustomerDetailsParameters searchParams) {
```

Comment:
Put `@ValidCustomerDetailsParameters` on the CustomerDetailsParameters class instad ?